### PR TITLE
Update compliance.md

### PR DESCRIPTION
**Pull request by ghnu**
jaimeo

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/intro/compliance.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Microsoft Managed Desktop has achieved the following certifications:
 - [Cloud Security Alliance (CSA) STAR certification](https://docs.microsoft.com/compliance/regulatory/offering-CSA-Star-Certification)
 - [Service Organization Controls (SOC) 1, 2, 3](https://docs.microsoft.com/compliance/regulatory/offering-SOC)
 - [Payment Card Industry (PCI) Data Security Standard (DSS)](https://docs.microsoft.com/compliance/regulatory/offering-PCI-DSS)
+- [Health Insurance Portability and Accountability Act (HIPAA)](https://docs.microsoft.com/en-us/compliance/regulatory/offering-hipaa-hitech)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/compliance.md]:
```suggestion
- [Health Insurance Portability and Accountability Act (HIPAA)](https://docs.microsoft.com/compliance/regulatory/offering-hipaa-hitech)
```

File: microsoft-365/managed-desktop/intro/compliance.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Microsoft Managed Desktop has achieved the following certifications:
...
+- [Information Security Registered Assessor Program (IRAP)](https://docs.microsoft.com/en-us/compliance/regulatory/offering-ccsl-irap-australia)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/compliance.md]:
```suggestion
- [Information Security Registered Assessor Program (IRAP)](https://docs.microsoft.com/compliance/regulatory/offering-ccsl-irap-australia)
```

File: microsoft-365/managed-desktop/intro/compliance.md [hunk 3]
```diff
@@ -40,4 +42,4 @@ You can find relevant information, including control and technical requirements,
 
 ## Shared responsibility
 
-Compliance for cloud services is a shared responsibility between cloud service providers and their customers. For more, see [Shared Responsibilities for Cloud Computing](https://gallery.technet.microsoft.com/Shared-Responsibilities-81d0ff91) in the TechNet Gallery.
+Compliance for cloud services is a shared responsibility between cloud service providers and their customers. For more, see [Shared Responsibilities for Cloud Computing](https://docs.microsoft.com/en-us/azure/security/fundamentals/shared-responsibility).
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/compliance.md]:
```suggestion
Compliance for cloud services is a shared responsibility between cloud service providers and their customers. For more information, see [Shared responsibility in the cloud](https://docs.microsoft.com/azure/security/fundamentals/shared-responsibility).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@ghnu

**Comment by yogkumgit** [comment]
@ghnu Please implement the feedback so that merge can be initiated. Thanks.

**Pull request merged**